overlays/DIMO/dimo.vspec

```diff
@@ -66,3 +66,73 @@ Vehicle.Powertrain.TractionBattery.Charging.AddedEnergy:
   datatype: float
   unit: kWh
   description: Amount of charge added to the high voltage battery during the current charging session, expressed in kilowatt-hours.
+
+Vehicle.ADAS.CruiseControl.CruiseFollowTime:
```

Comment:
https://github.com/COVESA/vehicle_signal_specification/commit/dcaf5c9c8dbe9ed8f649e246773566134d40eda5

Some inspiration (this was added after we forked)